diff --git a/cache/core/cache-effect.js b/cache/core/cache-effect.js
index 81ee54a670d1fb8abc9a5261b9b25b0f418960ea..ecc64c8a50da2b9db831f8246b42a213ea2dab28 100644
--- a/cache/core/cache-effect.js
+++ b/cache/core/cache-effect.js
@@ -94,7 +94,7 @@ function make(cache) {
 * This error is thrown when any cache operation (get, put, onMutate) fails.
 * The original error is available in the `cause` property.
 */
-var EffectCacheError = class extends Schema$1.TaggedErrorClass()("EffectCacheError", { cause: Schema$1.Unknown }) {
+var EffectCacheError = class extends Schema$1.TaggedError()("EffectCacheError", { cause: Schema$1.Unknown }) {
 	static [entityKind] = "EffectCacheError";
 };
 
diff --git a/effect-core/errors.js b/effect-core/errors.js
index 290da3a291c8919c16f1092c74e0a7f4a5feaf53..25f33b5473c016649c9370b681dc216697eb80a1 100644
--- a/effect-core/errors.js
+++ b/effect-core/errors.js
@@ -2,13 +2,13 @@ import { entityKind } from "../entity.js";
 import * as Schema$1 from "effect/Schema";
 
 //#region src/effect-core/errors.ts
-var EffectDrizzleError = class extends Schema$1.TaggedErrorClass()("EffectDrizzleError", {
+var EffectDrizzleError = class extends Schema$1.TaggedError()("EffectDrizzleError", {
 	message: Schema$1.String,
 	cause: Schema$1.Unknown
 }) {
 	static [entityKind] = "EffectDrizzleError";
 };
-var EffectDrizzleQueryError = class EffectDrizzleQueryError extends Schema$1.TaggedErrorClass()("EffectDrizzleQueryError", {
+var EffectDrizzleQueryError = class EffectDrizzleQueryError extends Schema$1.TaggedError()("EffectDrizzleQueryError", {
 	query: Schema$1.String,
 	params: Schema$1.Array(Schema$1.Any).pipe(Schema$1.mutable),
 	cause: Schema$1.Unknown
@@ -22,11 +22,11 @@ var EffectDrizzleQueryError = class EffectDrizzleQueryError extends Schema$1.Tag
 		Error.captureStackTrace(this, EffectDrizzleQueryError);
 	}
 };
-var EffectTransactionRollbackError = class extends Schema$1.TaggedErrorClass()("EffectTransactionRollbackError", {}) {
+var EffectTransactionRollbackError = class extends Schema$1.TaggedError()("EffectTransactionRollbackError", {}) {
 	static [entityKind] = "EffectTransactionRollbackError";
 	message = "Rollback";
 };
-var MigratorInitError = class extends Schema$1.TaggedErrorClass()("MigratorInitError", { exitCode: Schema$1.Literals(["databaseMigrations", "localMigrations"]) }) {
+var MigratorInitError = class extends Schema$1.TaggedError()("MigratorInitError", { exitCode: Schema$1.Literals(["databaseMigrations", "localMigrations"]) }) {
 	static [entityKind] = "MigratorInitError";
 };
 
